pkg/storage/storage.go

```diff
@@ -657,7 +657,11 @@ func (s *Module) checkAndResizeCache(cache filesystem.Volume, sizeMultiplier uin
 	if err != nil {
 		return errors.Wrap(err, "failed to check cache usage")
 	}
+	if usage.Size == 0 {
+		usage.Size = 5
```

Comment:
5 what? :D 
When i said set the size of the cache to 5. I meant 5gib. and that's not just a random number, it's the sizeMultiplier. 

But anyway, this is not exactly i wanted it to be done. The idea is if the cache.Size is not set (no limit) I wanted to make sure the `limit` is actually set first, before you even try to adjust.

```go
if useage.Size == 0 {
   usge.Size = max(sizeMultiplier, usage.Excl)
   if err := cache.Limit(usage.Size); err != nil {
       return err
   }
}
// ... rest of the function
```

Reply:
To test this, make sure your cache volume has no quota set then restart storaged. then the cache should have some limit